lib/filterItems.tsx

```diff
@@ -0,0 +1,6 @@
+export const FILTER_ITEMS = [
+  { label: "All", value: "all", isActive: true },
```

Comment:
Where did you got these genres? It seems hardcoded. You could make them dynamic, by getting the list from the json file. Each movie has `genre`. So the output would be:

```
const MOVIE_GENRES = ['Drama', 'Action', 'Crime', ....]
```

Reply:
Can I use an object instead of a string of an array?

Reply:
Yes you can :)